app/_hub/inigo/inigo/schemas/_index.json

```diff
@@ -0,0 +1,19 @@
+{
+  "fields": [
+    {
+      "consumer_group": {
+        "eq": null,
+        "type": "foreign",
+        "reference": "consumer_groups"
+      }
+    },
+    {
+      "config": {
+        "fields": [
```

Comment:
Looks like the config fields are missing here. Could you please add them, then base the example file off the basic config that the plugin expects?

The example file would be at `app/_hub/inigo/inigo/examples/_index.yml`.

Reply:
Our current plugin doesn't have any configuration, I wasn't sure how to represent that?

Reply:
Hmm. That's a good question. We haven't had that use case before, so I'm not sure that we've built in support for disabling the config key in the output (here: https://deploy-preview-7785--kongdocs.netlify.app/hub/inigo/inigo/configuration/). Let me look into that.